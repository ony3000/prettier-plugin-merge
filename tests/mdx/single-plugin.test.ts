import { testSnapshotEach } from '../adaptor';
import type { Fixture } from '../settings';
import { baseOptions } from '../settings';

const options = {
  ...baseOptions,
  parser: 'mdx',
};

const fixtures: Omit<Fixture, 'output'>[] = [
  {
    name: "(1) This plugin doesn't support mdx parser, so it leaves the output of each plugin as is.",
    input: `
\`\`\`jsx
export function Foo({ children }) {
  return (
    <div className="lorem ipsum dolor sit amet consectetur adipiscing elit proin ex massa hendrerit eu posuere eu volutpat id neque pellentesque">
      {children}
    </div>
  );
}
\`\`\`
`,
    options: {
      plugins: ['prettier-plugin-classnames'],
    },
  },
];

testSnapshotEach(fixtures, options);
