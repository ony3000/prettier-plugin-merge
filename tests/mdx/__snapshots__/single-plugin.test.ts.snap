// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'(1) This plugin doesn\\'t support mdx parser, so it leaves the output of each plugin as is.' > expectation 1`] = `
"\`\`\`jsx
export function Foo({ children }) {
  return (
    <div className="lorem ipsum dolor sit amet consectetur adipiscing elit proin ex massa hendrerit eu posuere eu volutpat id neque pellentesque">
      {children}
    </div>
  );
}
\`\`\`
"
`;
